README.md

```diff
@@ -183,3 +183,31 @@ The PR is the only place any team member is expected to consult for information
 [soliumrc]: ./.soliumrc.json
 [travisci]: https://travis-ci.org/
 [webpack]: https://webpack.js.org/
+
+
+### Managing internal and external dependencies
+* Internal dependencies:
+    1. Internal dependencies are dependencies which are under `@openst` project. 
+    2. Until the stable version is released, we should use version ranges to avoid any breaking change in newer minor versions.
+        Example:
+        ```
+               "dependencies": {
+                  "@openst/mosaic.js": ">=0.10.0 < 0.11.0"
+                }      
+        ``` 
+    3. When patch versions are released, then we should use tilde `~`.
+        ```
+               "dependencies": {
+                          "@openst/mosaic.js": "~0.1.0"
```

Comment:
This doesn't make any sense. You can also use the tilde operator in the example above, but how is it different here than the above `">=0.10.0 < 0.11.0"`?

The tilde operator could be a good idea actually (to replace the above statement, not in addition), but see review comment and read documentation. Caret is actually sufficient.

Reply:
I went through the documentation mentioned.
1. We can use caret until stable version is released. Currently, we have `0.10.0` version released. So use of caret will allow auto-update between the range `>=0.10.0 < 0.11.0`.
2. Once we have stable version, use of caret will lead to updating to a version having breaking changes. Consider we have released `1.2.3` version. Use of caret will allow auto-update to latest version between `>=1.2.3 <2.0.0`. There might be some breaking changes in the minor versions which we have to avoid to get propagated.
Let me know your thoughts.

Reply:
Very good 👍 
Once we have a stable 1.0.0 release, as per definition of [SemVer], there should not be any breaking changes in MINOR upgrades 😉 

[semver]: https://semver.org/